lib/base/json.cpp

```diff
@@ -392,6 +392,8 @@ template<bool prettyPrint>
 inline
 void JsonEncoder<prettyPrint>::Strng(String value)
 {
+	Utility::ValidateUTF8(value);
+
```

Comment:
... given that you're copying here during the function call due to its parameter by value?